pkg/component/kubeapiserver/monitoring.go

```diff
@@ -247,6 +253,8 @@ var (
 		monitoringMetricWatchCacheCapacity,
 		monitoringmetricApiserverCacheList,
 		monitoringmetricApiserverStorageList,
+		monitoringMetricApiserverRequestDurationSecondsCount,
+		monitoringMetricApiserverRequestDurationSecondsBucket,
```

Comment:
```suggestion
```

To follow the conventions this should also have been sorted alphabetically, but actually this is not needed because the `monitoringMetricApiserverRequestDurationSeconds` is a regular expression that allow lists these metrics as well.